config/loader_hcl2.go

```diff
@@ -0,0 +1,461 @@
+package config
+
+import (
+	"fmt"
+	"sort"
+	"strings"
+
+	gohcl2 "github.com/hashicorp/hcl2/gohcl"
+	hcl2 "github.com/hashicorp/hcl2/hcl"
+	hcl2parse "github.com/hashicorp/hcl2/hclparse"
+	"github.com/zclconf/go-cty/cty"
+)
+
+// zclConfigurable is an implementation of configurable that knows
```

Comment:
"zcl" still in comments from here down

Reply:
Thanks for catching these... there's a lot of this around still from the prototype, sadly. I'll fix these ones, and be on the look out for more as I keep working.